### PR TITLE
[Serve] Update min_replicas

### PR DESCRIPTION
**Pull request by MaoZiming**
Tested (run the relevant ones):

- [x] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

**Review by Michaelvll — APPROVED** [review]
Thanks for the fix @MaoZiming! Should we add a smoke test for the scaling up/down with the `update`?

**Comment by MaoZiming** [comment]
@Michaelvll Thanks. added a smoke test

File: examples/serve/http_server/task.yaml [hunk 1]
```diff
@@ -10,7 +10,8 @@ service:
   readiness_probe:
     path: /health
     initial_delay_seconds: 20
-  replicas: 2
+  replica_policy:
+    min_replicas: 2
```
Comment (Michaelvll) [inline examples/serve/http_server/task.yaml:14]: Do we want to still use `replicas: 2` for simplicity?

**Review by Michaelvll — APPROVED** [review]
LGTM

File: examples/serve/http_server/task.yaml (hunk 1 above)
Comment (MaoZiming) [reply examples/serve/http_server/task.yaml:14]: oops good catch

**Pull request merged**